Formula/hatari.rb

```diff
@@ -20,8 +16,15 @@ class Hatari < Formula
 
   # Download EmuTOS ROM image
   resource "emutos" do
-    url "https://downloads.sourceforge.net/project/emutos/emutos/0.9.6/emutos-512k-0.9.6.zip"
-    sha256 "2c7d57cac6792d0c7e921f9655f224b039402283dd24c894b085c7b6e9a053a6"
+    url "https://downloads.sourceforge.net/project/emutos/emutos/0.9.9.1/emutos-512k-0.9.9.1.zip"
+    sha256 "ab94cd249aebd7fb1696cbd5992734042450d8b96525f707e9ad8a2283185341"
+  end
+
+  if MacOS.verion <= :high_sierra
+    patch do
+      url "url du patch"
```

Comment:
You need to actually put the patch URL here, and its SHA-256 checksum below

Reply:
I waiting to have this url. I guess, I will have it when my pull request will validate by the owner of formula-patches repository. But, I can fill up the sha256, right now.